src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -1171,45 +1171,49 @@ public async Task TestFixAllOccurrences_CodeRefactoring_ContainingMember()
             var markup = @"
 class C1
 {
-    void M(bool x, int a, int b)
```

Comment:
Switched existing FixAll integration tests for refactorings to use `Convert to raw string` refactoring instead of `Invert conditional` refactoring.

Reply:
is the intent to remove fix-all-for-invert-if?  (if so, i approve of removing that).

Reply:
> is the intent to remove fix-all-for-invert-if?

Yes, I thought of doing it in a separate PR, but let me just push a commit to revert it here itself.